modules/concepts/hooks/list-of-hooks/action<DefinitionId>GridDataModifier.md

```diff
@@ -1,34 +1,30 @@
 ---
-menuTitle: action<DefinitionId>GridDataModifier
 Title: action<DefinitionId>GridDataModifier
 hidden: true
 hookTitle: 
 files:
-  - src/Core/Grid/GridFactory.php
+    -
+        url: 'https://github.com/PrestaShop/PrestaShop/blob/8.0.x/src/Core/Grid/GridFactory.php'
```

Comment:
One idea. Could we do something like this?

```
url: https://github.com/PrestaShop/PrestaShop/blob/8.0.x/
path: src/Core/Grid/GridFactory.php
```

url could be updated from one single place, path is unique, wdyt? are there any corner cases where this cannot work?